tracer/src/Datadog.Trace/RemoteConfigurationManagement/ConfigPath.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="ConfigPath.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Text.RegularExpressions;
+
+namespace Datadog.Trace.RemoteConfigurationManagement;
```

Comment:
As per the comment we received on the the open-sourcing PR, we should use block-scope namespace declarations for now, for consistency with the rest of the codebase.